metadata-schema/baseSchema.schema.json

```diff
@@ -24,65 +11,45 @@
         }
       ],
       "message": {
```

Comment:
I'm not sure I understand how this works. The keys `message` and `format` don't make sense given the it appears that these represent error messages, ie use `ERROR_...` Does it not make more sense to have:
```
...
"errorMessage": "ERROR_..."`
...
```

Reply:
this was was by the json schmea validator for the error message 
after looking at the documentation these can be defined in a separate properties file 
[https://github.com/networknt/json-schema-validator/blob/master/src/main/resources/jsv-messages.properties](https://github.com/networknt/json-schema-validator/blob/master/src/main/resources/jsv-messages.properties)
so removing all messages to make schema cleaner